src/lib/api/crew.ts

```diff
@@ -8,15 +8,9 @@ export async function getCrewMetaData({ workId }: { workId: string }) {
         .single();
 }
 
-export async function getCrews({
-    pageIndex,
-    pageSize,
-}: {
-    pageIndex: number;
-    pageSize: number;
-}) {
+export async function getCrews(pageIndex: number, pageSize: number) {
     return await supabaseBrowserClient
         .from('crews')
-        .select('id, root_work:root_id(id, title)', { count: 'exact' })
+        .select('id, root_work:root_id(id, title, description)', { count: 'exact' })
```

Comment:
```suggestion
        .select('id, root_work:root_id(id, title)', { count: 'exact' })
```

we don't want to fetch description when fetching many crews because it will increase our supabase pricing.

Reply:
we don't need to show desc on `/workspace` page

Reply:
okay